tools/budgets/src/budgets/check.py

```
@@ -0,0 +1,212 @@
+             f"{'段':<14}{'预算P95':>10}{'实际P95':>10}{'差值':>8}  状态"]
+    for row in result.rows:
+        delta = ("+" if row.delta > 0 else "") + fmt_ms(row.delta)
+        lines.append(f"{row.id:<14}{fmt_ms(row.budget_p95):>10}{fmt_ms(row.actual_p95):>10}"
+                     f"{delta:>8}  {row.status}")
+    lines.append("─" * 58)
+    lines.append(f"ΣP95={fmt_ms(result.total_actual)} 并行重叠={fmt_ms(result.overlap_ms)}"
+                 f" 有效总延迟={fmt_ms(result.effective_ms)} 总预算={fmt_ms(result.budget_ms)}"
+                 f" 超支={fmt_ms(result.over_by_ms)}")
+    if result.over_segments:
+        lines.append(f"超标段: {', '.join(result.over_segments)}")
+        lines.append(f"守恒校验违反：ΣP95−并行重叠={fmt_ms(result.effective_ms)}ms > 总预算"
```

Ruff (0.16.2)

[warning] String contains ambiguous `：` (FULLWIDTH COLON). Did you mean `:` (COLON)?

(RUF001)